common/Fabric-Install-Utilities.psm1

```diff
@@ -158,9 +158,6 @@ function Publish-WebSite($zipPackage, $appDirectory, $appName, $overwriteWebConf
         }
 
         if(!(Test-IsDirectory $itemTargetFilePath)){
-            if($itemTargetFilePath.EndsWith("web.config")){
-                $overwrite = $overwriteWebConfig
```

Comment:
overwriteWebConfig is completely ignored now, is there any case where web.config should not be overwritten now?

Reply:
I don't believe so - I think the change to reset secrets allows us to overwrite the configs at will on deployment.